src/index.d.ts

```diff
@@ -5208,52 +5208,123 @@ declare namespace admin.machineLearning {
    * Interface representing options for listing Models.
    */
   interface ListModelsOptions {
+    /**
+     * An expression that specifies how to filter the results.
+     *
+     * Examples:
+     *
+     * ```
+     * display_name = your_model
+     * display_name : experimental_*
+     * tags: face_detector AND tags: experimental
+     * state.published = true
+     * ```
+     *
+     * See https://firebase.google.com/docs/ml-kit/manage-hosted-models#list_your_projects_models
+     */
     filter?: string;
+
+    /** The number of results to return in each page. */
     pageSize?: number;
+
+    /** A token that specifies the result page to return. */
     pageToken?: string;
   }
 
   /** Response object for a listModels operation. */
   interface ListModelsResult {
+    /** A list of models in your project. */
     readonly models: Model[];
+
+    /** 
+     * A token you can use to retrieve the next page of results. If null, the
+     * current page is the final page.
+     */
     readonly pageToken?: string;
   }
 
   /**
-   * A TFLite Model output object
+   * A TensorFlow Lite Model output object
    */
   interface TFLiteModel {
+    /** The size of the model. */
     readonly sizeBytes: number;
 
+    /** The URI of the model in Cloud Storage. */
```

Comment:
This is true, but there is a bit of subtlety here too. Maybe we should mention something? I mean: The URI is more properly characterized as original source URI. This is the uri that the user supplied for create or update API calls, but once those calls have completed, the model file at that URI location can be modified or deleted without affecting the Firebase model. (We make our own copy of it during create/update).